src/pages/KadraVykhovnykiv/KVTable.tsx

```diff
@@ -102,10 +102,10 @@ export const KVTable = ({ current, searchData }: props) => {
   filteredData = filteredData.concat(
     data.filter(
       (item) =>
-        (item.user.firstName?.toLowerCase()?.includes(searchData) ||
-          item.user.lastName?.toLowerCase()?.includes(searchData)||
-          item.user.firstName?.includes(searchData) ||
-          item.user.lastName?.includes(searchData)) &&
+        ( item.user.firstName?.toLowerCase()?.includes(searchData.toLowerCase()) ||
+          item.user.lastName?.toLowerCase()?.includes(searchData.toLowerCase())||
+          (item.user.firstName+ ' ' + item.user.lastName)?.toLowerCase()?.includes(searchData.toLowerCase())||
```

Comment:
Please, remove unnecessary round brackets and add space near plus sign

Reply:
try alt+shift+f after removing unnecessary round brackets